movie/src/components/Main/Posters/Posters.tsx

```diff
@@ -0,0 +1,46 @@
+import React, { useEffect, useState } from "react";
+import Poster from "../Poster";
+import { ImovieItem, Tmovies } from "../../../interfaces/Poster.types";
+import { API_URL } from "../../../variables/api";
+
+const Posters: React.FC = () => {
+  const [movies, setMovies] = useState<Tmovies>([]);
+
+  useEffect(() => {
+    fetch(API_URL)
+      .then(res => res.json())
+      .then(data => {
+        setMovies(data.results);
+      });
+  }, []);
+
+  console.log(movies);
```

Comment:
Try to avoid every console information. We really need to incapsulate all in security reasons. Because your commit move to develop. Then deploy. Then it give a way for hack your app.
Best way - setup linter with erroring for every console.
Try to look at the chrome dev tools and use it instead of this.

Reply:
fixed